src/main/java/pl/pop/interview/master/answer/AnswerDTO.java

```diff
@@ -10,13 +10,13 @@
 public class AnswerDTO {
     private int id;
     private String question;
-    private String answer;
+    private boolean isCorrect;
```

Comment:
bardzo myląca nazwa.
nie wiadomo czy to jest treść odpowiedzi na pytanie: tak lub nie czy komentarz systemu czy nasza odpowiedź była poprawna czy nie. 
Zostałbym przy nazwie answer mimo że to boolean